bin/pytorch_inference/CCommandParser.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#ifndef INCLUDED_ml_torch_CCommandParser_h
+#define INCLUDED_ml_torch_CCommandParser_h
+
+#include <rapidjson/document.h>
+
+#include <functional>
+#include <iosfwd>
+#include <string>
+#include <vector>
+
+namespace ml {
+namespace torch {
+
+//! \brief
+//! Reads JSON documents from a stream calling the request handler
+//! for each parsed document.
+//!
+//! DESCRIPTION:\n
+//! Validation on the input documents is light. It is expected the input
+//! comes from another process which tightly controls what is sent.
+//! Input from an outside source that has not been sanitized should never
+//! be sent.
```

Comment:
This is saying to a hacker, "If you can manage to send dodgy input to this process we'll give you a shell prompt on the system."

I _think_ the input _is_ actually validated to the extent of preventing array bounds overwrites.  So instead the comment could be more along the lines of, "Validation exists to prevent memory violations from malicious input, but no more.  The caller is responsible for sending input that will not result in errors from libTorch and will produce meaningful results."

Reply:
👍